### DISCUSSION TITLE
I'm getting an error on H150i Elite

### DISCUSSION THREAD

**Discussion by HaidaLaOBere** [EvanMulawski/FanControl.CorsairLink#92] [General]
I installed the plugin as instructed and installed (I hope) corsiarsiusbxp.inf (it gave me a error "Certificate was explicitly revoked by it's issuer) 
After a while I got propped with this in corsairlink.log ( I keep getting the same error except for readCrcByte and readBufferCrcResult)

[CorsairLink.log](https://github.com/EvanMulawski/FanControl.CorsairLink/files/12501475/CorsairLink.log)

Can someone help me?

**Comment by EvanMulawski** [comment]
Hi @HaidaLaOBere, please try [v1.4.0-beta.5](https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.4.0-beta.5). This version resolves several issues with that device family.

**Reply by HaidaLaOBere** [reply]
ok will do

**Reply by HaidaLaOBere** [reply]
Hello @EvanMulawski well it doesn't work, I still got the massage and from what I can see it still shows that error

[CorsairLink.log](https://github.com/EvanMulawski/FanControl.CorsairLink/files/12502164/CorsairLink.log)

**Reply by EvanMulawski** [reply]
@HaidaLaOBere Moved to issue #93
